packages/databases-collections/src/collections-plugin.spec.tsx

```diff
@@ -59,7 +59,8 @@ describe('Collections [Plugin]', function () {
     cleanup();
   });
 
-  describe('with loaded collections', function () {
+  // TODO
```

Comment:
The table isn't rendering any rows in unit tests, so I haven't bothered trying to port these as the tests won't work until I find a workaround anyway.

Reply:
I just did as you typed - commit's just pushed.